General-workflow.md

```diff
@@ -1 +1,93 @@
+By entering a regex expression specifying a particular set of results, users can retrieve hits relevant to their investigation. These expressions can be specific, querying individual sentences or words, or more general, querying for a range of words, orthographical possibilities or structures using wildcards. Examples of queries used in MBG's research below along with descriptions of the intended target.
+
+ * *(Insert example queries here)
+ 
+The regex expressions can recognise any string of words and symbols (provided those symbols that are wildcards are 'escaped' first) and allows for underspecified items and alternatives. The more specific the query entered is, the less 'noise' will be retrieved in the results; the more general, the wider the scope will be, the query retrieving a wider range of possible results. Below find a list of useful, often used regex expressions.
+
+ * *(Insert regex key here)
+ 
+Queries are entered in the search box of the query pane.
+
+### **Query Pane**
+
+The query pane is the main window of Cosycat's research functions, allowing users to query the corpus, view retrieved hits in a paginated format, sort results, annotate queries, apply filters and change view settings. 
+
+* *(Insert image here)
+
+The query pane UI can be seen above: the interface is fairly straightforward with 5 primary elements as well as the menu bar. 
+
+1. **Corpus Selection:** This dropdown menu selects the corpus to be queried from the list available to the application.
+
+2. **Window and Size settings:** These dropdowns, as in the general settings menu, control the number of items around a hit in context (Window) and the number of hits per page (Size).
+
+3. **Searchbox:** This textbox is used to enter and search the regex expression query.
+
+4. **Pane-switch:** This button allows the user to switch between the query and annotation panes. When switched the panes are persistent, maintaining the status of hits, searches and marks.
+
+5. **Annotation/pencil icon:** This icon allows a query to be annotated and saved for later use.
+
+Once a search is conducted more options are revealed:
+
+* *(insert image here)*
+
+Here we see 6 further options, elucidated below:
+
+1. **Sort Criterion:** This option opens a set of further options, allowing a user to select sort criteria for the hits of a query. The criteria options are: 
+  * *:position*: match/left/right
+  * *:attribute*: word
+  * *:facet*: sensitive/insensitive
+Multiple sort criteria can be applied to the same query through the use of the *+* button; the *-* button removes these sort functions.
+
+2. **Pagination:** These options allow a user to move through the pages of results backward, foward and by specifying a page number.
+
+3. **Filters:** The filter option controls the filter options applied to the query, constraining its retrieval from the corpus to a set of data. The options here use sets based on the XML data in the corpus and include the following potential constraints:
```

Comment:
Since filters are actually automatically created based on the indexed structural attributes of the corpus (so, different corpora will have different filters), perhaps we should note this and also refer to the ones in the mbg corpus as example filters.